.github/workflows/mulled.yaml

```diff
@@ -42,6 +42,8 @@ jobs:
         with:
           path: .tox
           key: tox-cache-${{ runner.os }}-${{ steps.full-python-version.outputs.version }}-${{ hashFiles('galaxy root/requirements.txt') }}-mulled
+      - name: Install Apptainer's singularity
```

Comment:
Nice, then we can also run the singularity integration tests!

Reply:
https://github.com/galaxyproject/galaxy/pull/14922